core/math/plane.cpp

```diff
@@ -157,8 +154,13 @@ bool Plane::intersects_segment(const Vector3 &p_begin, const Vector3 &p_end, Vec
 /* misc */
 
 bool Plane::is_almost_like(const Plane &p_plane) const {
+	// TODO: Remove this in Godot 4.0.
```

Comment:
It's not exposed, so it can be removed already.

Reply:
Done.